leif/parse_slo_rules_test.go

```diff
@@ -0,0 +1,632 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package leif
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+	"time"
+)
+
+var defaultSLO = SLORule{AppliesTo: AppliesTo{Issues: true, PRs: false}, ComplianceSettings: ComplianceSettings{RequiresAssignee: false, Responders: Responders{Contributors: "WRITE"}}}
+var oneMin, _ = time.ParseDuration("1m")
+var oneHour, _ = time.ParseDuration("1h")
+var oneDay, _ = time.ParseDuration("24h")
+
+var syntaxError *json.SyntaxError
+var unmarshalTypeError *json.UnmarshalTypeError
+
+// func TestParsesSLORules(t *testing.T) {
```

Comment:
Is there a reason why these tests are commented out? They seem like valid tests that can be run.

Reply:
Sorry; I was testing the coverage without these (now that I think of it there is probably a better way to do that... 🤔). Uncommented